src/index.js

```diff
@@ -1,14 +1,22 @@
-import React from 'react';
-import ReactDOM from 'react-dom';
-import './index.css';
-import App from './App';
-import * as serviceWorker from './serviceWorker';
+import React from "react";
+import ReactDOM from "react-dom";
+import "./index.css";
+import App from "./App";
+import Home from "./components/sidebar";
+import * as serviceWorker from "./serviceWorker";
 
+const name = "John Doe";
+const streams = [
+  "Stream1",
+  "Stream Two",
+  "This Is A Long Stream Name",
+  "And It Goes Like This",
+];
 ReactDOM.render(
   <React.StrictMode>
-    <App />
+    <Home name={name} streams={streams} />
   </React.StrictMode>,
-  document.getElementById('root')
+  document.getElementById("root")
 );
```

Comment:
You can remove the Home component from here btw, let it be `<App />`. We'll add a higher-order component later that manages the UI.